docs/cloud-services-config.md

```diff
@@ -115,9 +115,92 @@ This command will start dev server on `http://localhost:9999`
 
 ### Making changes
 
-You can use the [CSC documentation](https://github.com/RedHatInsights/cloud-services-config#chromefed-modulesjson) for reference.
+These are how environments correlate with files
 
-## New feature outside of legacy cloud services config
+## Preview
+
+- stage-preview -> `/static/beta/stage`
+- prod-preview -> `/static/beta/prod`
+
+## Stable
+
+- stage-stable -> `/static/stable/stage`
+- prod-stable -> `/static/stable/prod`
+
+### Adding Config for New Apps
+
+To enable a new app in our environments, you need to create configuration for it in main.yml and in above mentioned environment directory. After that create a PR to merge it. Changes to stage are deployed automatically and for prod release tag @crc-platform-experience-services in #forum-consoledot-ui with the PR link on slack.
```

Comment:
This is not true. The main.yml exists only for API docs config. All modules are now configured via fed-modules.json